geode-pulse/src/main/java/org/apache/geode/tools/pulse/internal/service/ClusterDetailsService.java

```diff
@@ -44,14 +45,20 @@
 public class ClusterDetailsService implements PulseService {
 
   private final ObjectMapper mapper = new ObjectMapper();
+  private final Repository repository;
```

Comment:
```suggestion
  @Autowired
  private Repository repository;
```

Reply:
How strongly do you feel about this? The general (strong) consensus is to avoid field injection. And we've found that constructor injection makes it way easier to write unit tests (if we ever get around to writing unit tests).

Reply:
I only prefer field injection because it's less code. If this makes testing easier, I am OK with it.